app/services/task_service.rb

```diff
@@ -0,0 +1,72 @@
+class TaskService
+  attr_accessor :errors
+
+  def create(user, task_params)
+    begin 
+      task = Task.new(task_params)
+      task.assigner_user_id = user.id 
+      task.save!
+    rescue StandardError => error
```

Comment:
dont rescue StandardError, think what exceptions might come and only rescue those,